protocols/virtualfund/virtualfund_single_hop_test.go

```diff
@@ -172,26 +172,29 @@ func TestClone(t *testing.T) {
 	}
 }
 
-func collectPeerSignaturesOnSetupState(V *channel.SingleHopVirtualChannel, myRole uint, prefund bool) {
+func collectPeerSignaturesOnSetupState(v *channel.SingleHopVirtualChannel, myRole uint, prefund bool) *channel.SingleHopVirtualChannel {
```

Comment:
This is a nice workflow. As I understand it:
- cloning the test objective's channel
- manually progressing the cloned channel
- generating update events with the progressed, cloned channel
- applying the test objective's `Update` to the generated event

Probably this function wants a new name though - it no longer collects the state signatures onto the passed-in channel. Maybe something like `peerSignedClone`.